src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,41 @@
+import { NavLink, useLocation } from 'react-router-dom';
+import classes from './Navigation.module.scss';
+import React, { useCallback } from 'react';
+import { IconType } from 'react-icons';
+
+interface NavigationProps {
+  navItems: { name: string; path: string; icon?: IconType }[];
+}
+
+const Navigation: React.FC<NavigationProps> = ({ navItems }) => {
+  const location = useLocation();
+
+  const isNavLinkActive = useCallback(
+    (to: string) => {
+      return location.pathname.includes(to);
+    },
+    [location]
+  );
+
+  return (
+    <nav>
+      <ul className={`${classes['c-navigation-container']}`}>
```

Comment:
I just realised what you did, so `<nav>` tag should have clasName `c-navigation` and then `<ul>` tag should have className `c-navigation__container`